packages/beacon-node/test/unit/chain/opPools/aggregatedAttestationPool.test.ts

```diff
@@ -222,66 +213,162 @@ describe("AggregatedAttestationPool - Electra", () => {
     vi.clearAllMocks();
   });
 
-  it("Multiple attestations with same attestation data different committee", () => {
-    // Attestation from committee 0
-    const committeeBits0 = BitArray.fromSingleBit(MAX_COMMITTEES_PER_SLOT, 0);
-
-    const attestation0: Attestation<ForkPostElectra> = {
-      ...attestation,
-      aggregationBits: new BitArray(new Uint8Array(committeeLength / 8).fill(1), committeeLength),
-      committeeBits: committeeBits0,
-    };
-
-    // Attestation from committee 1
-    const committeeBits1 = BitArray.fromSingleBit(MAX_COMMITTEES_PER_SLOT, 1);
-
-    const attestation1: Attestation<ForkPostElectra> = {
-      ...attestation,
-      aggregationBits: new BitArray(new Uint8Array(committeeLength / 8).fill(1), committeeLength),
-      committeeBits: committeeBits1,
-    };
-    // Attestation from committee 2
-    const committeeBits2 = BitArray.fromSingleBit(MAX_COMMITTEES_PER_SLOT, 2);
-
-    const attestation2: Attestation<ForkPostElectra> = {
-      ...attestation,
-      aggregationBits: new BitArray(new Uint8Array(committeeLength / 8).fill(1), committeeLength),
-      committeeBits: committeeBits2,
-    };
-
-    // Attestation from committee 3
-    const committeeBits3 = BitArray.fromSingleBit(MAX_COMMITTEES_PER_SLOT, 3);
-
-    const attestation3: Attestation<ForkPostElectra> = {
-      ...attestation,
-      aggregationBits: new BitArray(new Uint8Array(committeeLength / 8).fill(1), committeeLength),
-      committeeBits: committeeBits3,
-    };
-
-    pool.add(attestation0, attDataRootHex, attestation0.aggregationBits.getTrueBitIndexes().length, committees[0]);
-
-    pool.add(attestation1, attDataRootHex, attestation1.aggregationBits.getTrueBitIndexes().length, committees[1]);
+  const testCases: {
+    name: string;
+    // item is is for committee i, which contains array of attester indices that's not seen (seen by default)
+    notSeenInStateByCommittee: number[][];
+    // item i is for committee i, each item is number[][] which is the indices of validators not seen by the committee
+    // each item i also decides how many attestations added to the pool for that committee
+    attParticipationByCommittee: number[][][];
+    // expected committeeBits of packed attestations, item 0 is for returned attestation 0, ...
+    packedCommitteeBits: number[][];
+    // expected aggregationBits of packed attestations: item 0 is for returned attestation 0, ...
```

Comment:
I think this comments make `packedAggregationBits` sounds like it represents bit content like [0,1,0,1,1...]. But it actually represents the length of aggregation bits. 

And unless we are actually checking on the aggregation bit content, this doesn't serve too much of a purpose as 
```
packedAggregationBits[i] = packedCommitteeBits[i].length * committeeLength
```
always hold true

Reply:
I changed the name to `packedAggregationBitsLen` + add `packedAggregationBitsUint8Array` to verify its content